### PR TITLE
A bunch of fixes

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:
CL: [Fix] fixes version number not showing up on the bottom left
CL: [Fix] fixes Security console having white buttons
CL: [Fix] Fixes issues with shuttles ramming into station when docking
CL: [Fix] fixed some handling of cable placement visualization
CL: [Improvement] A minimum radius for asteroids to spawn in has been set, preventing them from spawning ontop of the station.
CL: [Fix] fixes pipe vision still being visible after being kicked out of disposals
CL: [Improvement] optimises some code related to research UI, teleportation and other things
CL: [Fix] fixes RCS not working sometimes when connected to another shuttle

### CODE REVIEW

File: UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs [hunk 1]
```diff
@@ -192,12 +185,10 @@ public OrientationEnum TargetOrientation
 
 	public bool IsMoving = false;
 
-	public readonly UnityEvent OnStartMovement = new UnityEvent();
-	public readonly UnityEvent OnStopMovement = new UnityEvent();
-
-	public readonly UnityEvent OnRotate = new UnityEvent();
-
-	public readonly UnityEvent OnRotate90 = new UnityEvent();
+	public event Action OnStartMovement;
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs:188]: Why change this? Keep them as unity events.

File: UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs [hunk 2]
```diff
@@ -304,23 +290,26 @@ public void Awake()
...
 
+		SetGizmoPosition(currentLocalPivot);
 	}
 
 	public void OnDestroy()
 	{
+		OnRotate = null;
+		OnRotate90 = null;
+		OnStartMovement = null;
+		OnStopMovement = null;
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs:312]: `RemoveAllListners()` exists on unity events

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs:312]: It is no longer unity events is c# events

File: UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs:188]: they were 4 times slower when adding and removing stuff

**Review by MaxIsJoe — APPROVED** [review]

**Pull request merged**